index.html

```diff
@@ -0,0 +1,143 @@
+                <a href=""><img class="blackHeaderIcon" src="./images/twitterIcon.png" alt=""></a>
+            </div>
+            <a href=""><div class="blackHeaderOption">English</div></a>
+            <a href=""><div class="blackHeaderOption">My Page</div></a>
+            <a href=""><div class="blackHeaderOption">Logout</div></a>
+        </div>
+        <div id="whiteHeader">            
+            <a href="./index.html"><img id="logoHeader" src="./images/logoOrangeMx-removebg-preview.png" alt="logo"></a>
+            <div id="menuOptions">
+                <a class="menuOptionsLinks" href="./about.html"><div>About</div></a>
+                <a class="menuOptionsLinks" href="./index.html"><div>Event</div></a>
+                <a class="menuOptionsLinks" href="#programDetails"><div>Join</div></a>
+                <a class="menuOptionsLinks sponsorMobile" href="./about.html#footerOne"><div>Sponsor</div></a>
+                <a class="menuOptionsLinks sponsorDesk" href="#footerOne"><div>Sponsor</div></a>
+                <a class="menuOptionsLinks" href="#"><div>News</div></a>
+                <a class="menuOptionsLinks" href="#"><div>CC Campaign</div></a>
+            </div>
+        </div>
+    </header>
+    <main>
+                                <!-- Main Section -->
+        <section class="main">
+            <div id="mobileTopContainer">
+                <nav class="hamburgerMenu">
+                    <span class="bar bar1"></span>
+                    <span class="bar bar2"></span>
+                    <span class="bar bar3"></span>
+                </nav>
+                <div id="firstMain"></div>
+            </div>
+            <!-- ---- -->
+                <img class="adornIcon taco" src="./images/iconsMxImages/6bigTaco-removebg-preview.png" alt="">
+                <img class="adornIcon hat" src="./images/iconsMxImages/2hat-removebg-preview.png" alt="">           
+                <img class="adornIcon pinata" src="./images/iconsMxImages/3pin-removebg-preview.png" alt="">
+                <img class="adornIcon guacamole" src="./images/iconsMxImages/7guacamole-removebg-preview.png" alt="">
+            <!-- --- -->
+            <div id="textMain">
+                <div id="helloAndEventContainer">
+                    <div id="hello">"Hello! Food Lovers"</div>
+                    <h1 id="event">INTERNATIONAL MEXICAN FOOD CONVENTION 2023</h1>
+                </div>
+                <div id="description">
+                    <p id="descriptionParagraph">A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries     is taking place in October, in Korea.</p>
+                </div>
+                <div id="dateAndPlaceContainer">
+                    <div id="date">2023.10.15(THU) ~ 16(FRI)</div>
+                    <div id="place">@ National Museum of Karea, Art Center Nabi and more</div>
+                </div>
```

Comment:
<img width="541" alt="image" src="https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/84156714/3cf07542-dfb3-473d-a498-f776f8ab1150">

In the above image which is a screenshot of your project, you can see that some text are hidden and not readable. KIndly fix this.

Reply:
[textshowing.webm](https://github.com/AdrianScharff/1st-Capstone-project-MexicanFood/assets/134537383/bfcf4ace-c033-4348-9553-52f5bbe680f9)
I'm sorry @bobb-Rob but I don't see any screen size by which the text is hidden as in your picture, as I show in the video the text is showing normally in all sizes